README.md

```diff
@@ -338,13 +338,19 @@ Example Script: `/build/ffmpeg_extra.sh` for `ffmpeg-git`
 ``` bash
 #!/bin/bash
 _pre_configure(){
+    #
     # Apply a patch from ffmpeg's patchwork site.
     do_patch "https://patchwork.ffmpeg.org/patch/12563/mbox/" am
+    #
+    # Apply a local patch inside the directory where is "ffmpeg_extra.sh"
+    patch -p1 -i "../ffmpeg-0001-my_patch.patch"
```

Comment:
I'd prefer not using relative paths at all, especially since this isn't the same for every `_pre_configure`.
Better to just use `/build` or `$LOCALBUILDDIR`, same as the suite.

Reply:
Ummm... I need to check it!